### PR TITLE
Update dg

### PR DESCRIPTION
**Pull request by LeeMingDe**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -290,6 +290,43 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+2. User inputs the new information about the patient.
+3. System edits the patient information to the new information.
+4. System saves the new patient information.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. System realises that no optional fields are inputted.
+
```
Comment (itssodium) [inline docs/DeveloperGuide.md:306]: If I'm not wrong the textbook didn't have these white spaces.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -290,6 +290,43 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+1. User requests to search patients with a criteria.
+2. System search the patients with the inputted criteria.
+3. System shows the search results.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. System realises that no such patient is recorded
+
```
Comment (itssodium) [inline docs/DeveloperGuide.md:323]: Same problem as above.

**Review by itssodium — APPROVED** [review]
Other than the small details, lgtm!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (LeeMingDe) [reply docs/DeveloperGuide.md:306]: whoops, let me rectify that

**Review by raymondge — APPROVED** [review]
LGTM

**Pull request merged**